pycheribuild/projects/disk_image.py

```diff
@@ -1137,6 +1183,22 @@ def __init__(self, *args, **kwargs):
         self.minimum_image_size = "256m"  # let's try to shrink the image size
 
 
+class BuildCheriBSDZFSDiskImage(BuildCheriBSDDiskImage):
+    target = "disk-image-zfs"
```

Comment:
I don't think we want a whole new set of targets (then you'll want freebsd- and freebsd-default-options- ones too...), just make it an option for disk-image

Reply:
I was wondering if we might end up wanting to be able to produce both in CI at some point, but I'd forgetting we've have to deal with all the other cases

Reply:
I've switch to an option. The one ugly bit is that it doesn't seem to interact well with _AdditionalFileTemplates and I ended up adding a new method for zfs.